devnet/node.yaml

```diff
@@ -121,7 +121,7 @@ spec:
             # - --terraLCD
             # - http://terra-terrad:1317
             # - --terraContract
-            # - terra18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5
```

Comment:
there are few copies of this address in the codebase remaining that we should review before merging

Reply:
The two `migrate.js` files are very old (seem like they're there for posterity) and the one in `testing/contract-integrations` isn't used. :shrug:

Reply:
should we just delete them then?

Reply:
Maybe, but I added the change to https://github.com/wormhole-foundation/wormhole/pull/3594 for now. Didn't want to hold up review of this.